src/pages/VenuePage/VenuePage.tsx

```diff
@@ -49,6 +48,7 @@ import { updateTheme } from "./helpers";
 import "./VenuePage.scss";
 
 import Login from "pages/Account/Login";
+import { useRelatedVenues } from "hooks/useRelatedVenues";
```

Comment:
Sort with other `hooks/*`

Reply:
import removed in 4908ac4abd39131d79fdaedea96781e81ebae0cb